### PR TITLE
Alerting: temporarily add old ng routes

### PR DESCRIPTION
**Pull request by domasx2**
Makes "old" NG alerting pages available at `/alerting/ng/new` and `/alerting/ng/list` so they can be tested side by side with unified alerting

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by kylebrandt — APPROVED** [review]
This helps, one other that would be helpful for interim is to also temporary restore the ng edit view, something like `alerting/ng/rFUGnwlMz/edit` (don't really care if the list for ng (not dashboard) alerts actually has the correct links or not.

File: public/app/routes/routes.tsx [hunk 1]
```diff
@@ -383,10 +389,17 @@ export function getAppRoutes(): RouteDescriptor[] {
...
     },
+    {
+      path: '/alerting/ng/new',
+      pageClass: 'page-alerting',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "NgAlertingPage"*/ 'app/features/alerting/NextGenAlertingPage')
+      ),
+    },
     {
       path: '/alerting/:id/edit',
```
Comment (peterholmberg) [inline public/app/routes/routes.tsx:404]: need to add one for ng-edit too

**Review by peterholmberg — CHANGES_REQUESTED** [review]

File: public/app/routes/routes.tsx (hunk 1 above)
Comment (domasx2) [reply public/app/routes/routes.tsx:404]: Done! Updated links as well in the "old" pages

**Review by peterholmberg — APPROVED** [review]
Great 😄

**Pull request merged**